### PR TITLE
Create admin-roles.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Docs for: https://github.com/mattermost/mattermost-server/pull/14960

### CODE REVIEW

**Comment by justinegeffen** [comment]
Docs for: https://github.com/mattermost/mattermost-server/pull/14960.

**Comment by justinegeffen** [comment]
@mkraft, some questions around this:
1. Can System Admins assign other areas of the System Console to specific roles (other than the default ones)? 
2. How do System Admins unassign roles for example if a user leaves the organization? 
3. These commands aren't currently in the mmctl docs - is that by design?

File: source/deployment/admin-roles.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
+=============================
+Configuring Admin Roles (E20)
```
Comment (wiersgallak) [inline source/deployment/admin-roles.rst:2]: Other features don't include a verb in the title.  Thinking this should be "Additional System Admin Roles" or something to keep consistent

File: source/deployment/admin-roles.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+=============================
+
+From v5.28, System Admins can assign roles to members, allowing them access only to designated areas of the System Console. This allows other members of your organization to perform certain administrative tasks without providing access to all administrative capabilities.
```
Comment (wiersgallak) [inline source/deployment/admin-roles.rst:5]:
```suggestion
From v5.28, System Admins can assign roles to members, allowing them access only to designated areas of the System Console. This allows other members of your organization to perform certain administrative tasks without providing access to all system administrative capabilities.
```

File: source/deployment/admin-roles.rst (hunk 1 above)
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:2]:
```suggestion
Additional System Admin Roles (E20)
```

File: source/deployment/admin-roles.rst [hunk 3]
```diff
@@ -0,0 +1,120 @@
+=============================
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:1]:
```suggestion
==================================
```

File: source/deployment/admin-roles.rst [hunk 4]
```diff
@@ -0,0 +1,120 @@
+==================================
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:1]:
```suggestion
====================================
```

File: source/deployment/admin-roles.rst [hunk 5]
```diff
@@ -0,0 +1,120 @@
+====================================
+Additional System Admin Roles (E20)
+=============================
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:3]:
```suggestion
====================================
```

File: source/deployment/admin-roles.rst (hunk 1 above)
Comment (justinegeffen) [reply source/deployment/admin-roles.rst]: Thanks for picking that up, absolutely right.

**Review by wiersgallak — APPROVED** [review]
Thanks!

File: source/deployment/admin-roles.rst [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No. System Managers can't elevate their role, and aren't able to elevate other members' roles.
+
+Can any of the new roles view API keys/passwords or other sensitive information within the System Console (such as SMTP, AWS, Elastic Search)?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No, password information is only visible to System Admins and is obfuscated for other roles.
+
+If download links for compliance exports are enabled in the System Console, can a Read Only Admin download the reports?
```
Comment (amyblais) [inline source/deployment/admin-roles.rst:87]: This was reverted from v5.28 cc @wiersgallak

Comment (justinegeffen) [reply source/deployment/admin-roles.rst]: Removed, thanks @amyblais.

**Review by amyblais — APPROVED** [review]

**Comment by justinegeffen** [comment]
@mkraft, reminder on this review. :)

**Review by thefactremains — APPROVED** [review]

**Pull request merged**